### PR TITLE
OOP school library: set up associations

### PR DESCRIPTION
**Pull request by Hamzaoutdoors**
# In this milestone I completed the following to practice OOP Associations 💪🏼  : 

- Created  `Classroom` class to practice *Aggregation* relationship (`has-many`/`belongs-to` between `Student` and `Classroom`)
- Created `Book` class to practice many to many *Association* relationship
- Created `Rental` class (`has-many-through` between `Person` and `Book`)

##  Build with : 

  - Ruby 💎 💎 💎 

------

_THANKS FOR YOUR FEEDBACK 💟_

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @Hamzaoutdoors, 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: library/book.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+require './library/rental'
+
+# Book class
+
+class Book
+  attr_reader :rentals
```
Comment (mke2111) [inline library/book.rb]: - [x] The rentals need to have `attr_accessor` since they both have to read and write each rental

Comment (Hamzaoutdoors) [reply library/book.rb]: Yes I missed that, thank you for your feedback 👍🏼

Comment (henatan99) [reply library/book.rb]: This issue has been addressed well. Great job! :clap: :clap: :clap:

**Review by henatan99 — APPROVED** [review]
Hi @Hamzaoutdoors , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: library/book.rb (hunk 1 above)
Comment (Hamzaoutdoors) [reply library/book.rb]: Thank you @henatan99, @mke2111

**Pull request merged**